apps/panel/src/components/molecules/webhook-add/WebhookAdd.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  BooleanInput,
+  Button,
+  FlexElement,
+  Icon,
+  Select,
+  StringInput,
+  TextAreaInput
+} from "oziko-ui-kit";
+import "oziko-ui-kit/dist/index.css";
+
+const WebhookAdd = () => {
+  return (
+    <FlexElement direction="vertical" gap={10} dimensionX={"fill"}>
+      <StringInput label="Name" onChange={() => {}} />
+      <StringInput label="Url" onChange={() => {}} />
+      <TextAreaInput placeholder="{{{ toJSON this }}}" />
+      <Select value={"Collection*"} options={[]} onChange={() => {}} />
+      <Select
+        value={"Type*"}
+        options={["INSERT", "UPDATE", "DELETE", "REPLACE"]}
+        onChange={() => {}}
+      />
+      <BooleanInput label="Active" onChange={() => {}} />
+      <FlexElement dimensionX={"fill"} alignment="rightCenter">
+        <Button onClick={() => {}}>
+          <Icon name="save"></Icon>
```

Comment:
use the self-closing format
```suggestion
          <Icon name="save" />
```